pkg/query/operators.go

```diff
@@ -33,6 +33,11 @@ const (
 	InOperator inOperator = "in"
 	// NotInOperator takes two operands and tests if the left is not contained in the right
 	NotInOperator notInOperator = "notin"
+	// NotExistsOperator receives a sub-query as single left-operand and checks the sub-query for rows existence, and if the NotExistsOperator receives a sub-query as single left-operand and checks the sub-query for rows existence,
+	// and if there are no rows then it are no rows then it will return TRUE, otherwise FALSE.
+	NotExistsOperator notExistsOperator = "notexists"
```

Comment:
maybe these operators can be private? they are only used in the package, and do not make much sense without the wrapping ByIDExist

Reply:
now i see they are used also from other packages, so it won't be that easy..

Reply:
Yes I totally agree.
Initially I wanted to make them private but that would've complicate the code so I think it would be better this way.

Reply:
After discussing over this issue, it has been decided to rename "NotExistsOperator" and "ExistsOperator" to NotExistsSubquery and ExistsSubquery in order to distinguish the aforementioned from the rest of the operators as these aren't "regular" operators.